go/src/koding/kites/kloud/kloud/kloud.go

```diff
@@ -30,6 +31,7 @@ import (
 	"koding/kites/kloud/terraformer"
 	"koding/kites/kloud/userdata"
 	"koding/remoteapi"
+	"koding/socialapi"
```

Comment:
can we rename socialapi to api or clients? socialapi is one of the service name and we have the same package path?